workflows/proteomics/metaquantome-function/README.md

```diff
@@ -0,0 +1,4 @@
+# metaQuantome 2: Function
```

Comment:
That's a strange title. Sure there is a tutorial with that name, but this is not the gtn repo and it won't make any sense without that context.